### ISSUE TITLE
v1.2.0 of terraform not available via apt repository

### ISSUE THREAD

**Issue by jnoynaert** [hashicorp/terraform#31104]
It looks like no one has cut a release of v1.2 to apt yet.

```bash
> sudo apt update
> apt policy terraform
terraform:
  Installed: 1.1.9
  Candidate: 1.1.9
  Version table:
 *** 1.1.9 500
        500 https://apt.releases.hashicorp.com buster/main amd64 Packages
        100 /var/lib/dpkg/status
     1.1.8 500
        500 https://apt.releases.hashicorp.com buster/main amd64 Packages
     1.1.7 500
        500 https://apt.releases.hashicorp.com buster/main amd64 Packages
     1.1.6 500
        500 https://apt.releases.hashicorp.com buster/main amd64 Packages
     1.1.5 500
        500 https://apt.releases.hashicorp.com buster/main amd64 Packages

```

**Comment by crw** [comment]
Thanks for this report. This should be resolved for 1.2.0 and subsequent releases. Thanks again!

**Comment by crw** [comment]
No, not to be expected, the release process was not completely fixed. 1.2.1 should be up now, and we are hoping that we've shaken out the last issue in the release process.
